src/text_file_functions.cpp

```diff
@@ -278,6 +278,16 @@ QVector<server_type> AOApplication::read_serverlist_txt()
         f_server.ip = line_contents.at(0);
         f_server.port = line_contents.at(1).toInt();
         f_server.name = line_contents.at(2);
+
+        if (line_contents.size() >= 4) {
+          QString socket_type = line_contents.at(3).toLower();
+          if (socket_type == "ws" || socket_type == "tcp") {
+            f_server.socket_type = to_connection_type.value(socket_type);
+          }
+          else {
+            qWarning() << "Serverlist.txt - Unknown socket type: " << socket_type;
+          }
+        }
```

Comment:
Fails to support colons in server names.

Reply:
Latest changes now support it : 
![image](https://user-images.githubusercontent.com/62221668/172025671-fce81c6b-fe14-45c8-9065-3f16c8db312e.png)